ui/components/app/app-header/app-header.component.js

```diff
@@ -110,12 +110,16 @@ export default class AppHeader extends PureComponent {
       disabled,
       onClick,
     } = this.props;
+    const pathname = history?.location?.pathname;
```

Comment:
```suggestion
```

Reply:
fixed in 66ce7f463bb930ba2978b71767c297777aaa69b9

Reply:
- Verified